### PR TITLE
Streamer integrated

### PR DESCRIPTION
**Pull request by aeyakovenko**
integrated into skel/stub

### CODE REVIEW

File: src/accountant_skel.rs [hunk 1]
```diff
@@ -55,21 +61,63 @@ impl AccountantSkel {
...
+        let msgs = r_reader.recv_timeout(timer)?;
+        let msgs_ = msgs.clone();
+        let msgs__ = msgs.clone();
+        let rsps = streamer::allocate(recycler.clone());
+        let rsps_ = rsps.clone();
+        let l = msgs__.read().unwrap().packets.len();
+        rsps.write().unwrap().packets.resize(l, streamer::Packet::default());
+        let mut num = 0;
+        let mut ursps = rsps.write().unwrap();
+        for packet in msgs.read().unwrap().packets {
```
Comment (aeyakovenko) [inline src/accountant_skel.rs]: @garious , cant figure this one out 
```
   Compiling silk v0.4.0 (file:///Users/anatolyy/silk)                                                                                                        
error[E0507]: cannot move out of borrowed content                                                                                                             
  --> src/accountant_skel.rs:79:23                                                                                                                            
   |                                                                                                                                                          
79 |         for packet in msgs.read().unwrap().packets {                                                                                                     
   |                       ^^^^^^^^^^^^^^^^^^^^ cannot move out of borrowed co                                                                                
ntent                                                                                                                                                         
                                                                                                                                                              
error: aborting due to previous error                                                                                                                         
                                                                                                                                                              
error: Could not compile `silk`.                                                                                                                              
warning: build failed, waiting for other jobs to finish...                                                                                                    
error[E0507]: cannot move out of borrowed content                                                                                                             
  --> src/accountant_skel.rs:79:23                                                                                                                            
   |                                                                                                                                                          
79 |         for packet in msgs.read().unwrap().packets {                                                                                                     
   |                       ^^^^^^^^^^^^^^^^^^^^ cannot move out of borrowed co                                                                                
ntent                                                                                                                                                         
                                                                                                                                                              
error: aborting due to previous error                                                                                                                         
                                                                                                                                                              
error: Could not compile `silk`.   
```

**Comment by garious** [comment]
Can you rebase on master?

File: src/accountant_skel.rs (hunk 1 above)
Comment (aeyakovenko) [reply src/accountant_skel.rs]: `packets.iter()` fixes it.  error was just weird

**Comment by garious** [comment]
What's up with CI? Can we get this in tonight?

**Pull request merged**